src/client/filehandler/FileContainer.tsx

```diff
@@ -36,31 +37,46 @@ export const FileContainer: React.FunctionComponent<Props> = (props: Props) => {
 
     const handleFileDeletion = async (fileID: string) => {
         await deleteFile({
-            sid: props.sessionID,
+            sid: props.id,
             fileId: fileID,
             uid: props.userID,
         });
-        props.socket.emit(FileUploadEvent.FILE_DELETED, props.sessionID);
-        props.updateFiles({ sid: props.sessionID, roomType: props.roomType });
+        props.socket.emit(FileUploadEvent.FILE_DELETED, props.id);
+        props.updateFiles({
+            id: props.id,
+            roomType: props.roomType,
+            fileUploadType: FileUploadType.DOCUMENTS,
+        });
     };
 
     const updateFileList = React.useCallback(() => {
-        props.updateFiles({ sid: props.sessionID, roomType: props.roomType });
+        props.updateFiles({
+            id: props.id,
+            roomType: props.roomType,
+            fileUploadType: FileUploadType.DOCUMENTS,
+        });
     }, [props]);
```

Comment:
`props` as dependencies? Object comparison is always false, destructure it and pass in the values individually.